pandas/tests/window/moments/test_moments_ewm.py

```diff
@@ -263,49 +265,68 @@ class TestEwmMomentsConsistency(ConsistencyBase):
     def setup_method(self, method):
         self._create_data()
 
-    def test_ewmcov(self):
-        self._check_binary_ew("cov")
+    def _create_binary_ew_data(self):
+        A = Series(randn(50), index=np.arange(50))
+        B = A[2:] + randn(48)
+
+        A[:10] = np.NaN
+        B[-10:] = np.NaN
+        return A, B
+
+    @pytest.mark.parametrize("min_periods", [0, 1, 2])
+    def test_ewmcov(self, min_periods):
+        A, B = self._create_binary_ew_data()
+
+        self._check_binary_ew(name="cov", A=A, B=B)
+        self._check_binary_ew_min_periods("cov", min_periods, A, B)
 
     def test_ewmcov_pairwise(self):
         self._check_pairwise_moment("ewm", "cov", span=10, min_periods=5)
 
-    def test_ewmcorr(self):
-        self._check_binary_ew("corr")
+    @pytest.mark.parametrize("min_periods", [0, 1, 2])
+    def test_ewmcorr(self, min_periods):
+        A, B = self._create_binary_ew_data()
+
+        self._check_binary_ew(name="corr", A=A, B=B)
+        self._check_binary_ew_min_periods("corr", min_periods, A, B)
 
     def test_ewmcorr_pairwise(self):
         self._check_pairwise_moment("ewm", "corr", span=10, min_periods=5)
 
-    def _check_binary_ew(self, name):
+    def _check_binary_ew(self, name, A, B):
```

Comment:
make this a free function, put in common.py

Reply:
changed